.vib/wordpress/goss/vars.yaml

```diff
@@ -0,0 +1,12 @@
+wordpressEmail: test_user_email@email.com
+service:
+  ports:
+    https: 443
```

Comment:
We can use `443 + 1` here as we do with other ports to check customization. I think it'll work 😁

Reply:
I left it as `443` because the cypress action is running using `HTTPS` and modifying it breaks the action (as using `http != 80` does if it uses HTTP). I have removed the https test in goss due to redundancy.